docs/source/reference.rst

```diff
@@ -3,6 +3,7 @@ Reference
 
 .. toctree::
 
+   manindex.rst
```

Comment:
This seems to be a duplicate of reference, just remove the page instead of including it here.